tools/stimulus/types-RR.yaml

```
@@ -291,6 +291,12 @@ EXTRA:
     CALL: {}
     HEADER: ~
 
+CLOSURE:
+    CALL: '%I%'
+
+MOTIFS_HANDLER_FUNC:
+    CALL: '%I%'
+
```

Also no longer needed:

```suggestion
MOTIFS_HANDLER_FUNC:
    CALL: '%I%'

```